packages/optimistic-oracle-v2/src/utils/helpers/optimisticOracle.ts

```diff
@@ -8,6 +10,13 @@ export function getOrCreateOptimisticPriceRequest(
 
   if (request == null && createIfNotFound) {
     request = new OptimisticPriceRequest(id);
+    request.identifier = "";
```

Comment:
Also we need to initialise the identifier and all non-nullable fields otherwise we get errors.